UPGRADING.md

```diff
@@ -0,0 +1,12 @@
+# Upgrading the Dropbox SDK
+
+This document is designed to show you how to upgrade to the latest version of the SDK accomodating any breaking changes introduced by major version updates.  If you find any issues with either this guide on upgrading or the changes introduced in the new version, please see [CONTRIBUTING.md][contributing]
```

Comment:
This will need a contributing link similar to the code of conduct at the bottom. Otherwise the contributing link won't go anywhere